docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -222,6 +222,18 @@ GET /druid/indexer/v1/supervisor/<supervisorId>/history
 ```
 Returns an audit history of specs for the supervisor with the provided ID.
 
+#### Reset Supervisor
+```
+POST /druid/indexer/v1/supervisor/<supervisorId>/reset
+```
+The indexing service keeps track of the latest persisted Kafka offsets in order to provide exactly-once ingestion
+guarantees across tasks. Subsequent tasks must start reading from where the previous task completed in order for the
+generated segments to be accepted. If the messages at the expected starting offsets are no longer available in Kafka
+(typically because the message retention period has elapsed) the supervisor will refuse to start and in-flight tasks
+will fail. This endpoint can be used to clear the stored offsets which will cause the supervisor to start reading from
+either the earliest or latest offsets in Kafka (depending on the value of `useEarliestOffset`). **Use of this endpoint
+will result in data loss or data duplication.**
```

Comment:
Can this endpoint be used while the supervisor is running?

Can this be used while the supervisor is *not* running?